src/main/java/org/elasticsearch/action/admin/cluster/state/ClusterStateResponse.java

```diff
@@ -55,7 +55,7 @@ public ClusterName getClusterName() {
     public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
         clusterName = ClusterName.readClusterName(in);
-        clusterState = ClusterState.Builder.readFrom(in, null, clusterName);
+        clusterState = ClusterState.Builder.readFrom(in, null);
```

Comment:
maybe it's more intuitive to have a `readFrom(StreamInput input)` that delegates than passing null here?